### DISCUSSION TITLE
Improve accessibility - captions

### DISCUSSION THREAD

**Discussion by lalong13** [TanStack/table#5296] [Ideas]
Would it be possible to add the ability to set captions on tables?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption

> A caption functions like a heading for a table. Most screen readers announce the content of captions. Captions help users to find a table and understand what it’s about and decide if they want to read it. If the user uses “Tables Mode”, captions are the primary mechanism to identify tables. The caption is provided by the <caption> element.

from: https://www.w3.org/WAI/tutorials/tables/caption-summary/

I don't think this would take much effort.

**Comment by KevinVandy** [comment]
Markup is not the responsibility of TanStack Table. What's stopping you from adding a caption while using TanStack Table?
